releasenotes/notes/add-synth-circuit-from-stabilizer-list-4cf9cfa01bbc7ddf.yaml

```diff
@@ -0,0 +1,9 @@
+---
+features:
+  - |
+    Add :func:`qiskit.synthesis.synth_circuit_from_stabilizers` function that, given stabilizers,
+    returns a circuit that outputs the state stabilized by the stabilizers.
+
+  - |
+    Add :meth:`qiskit.quantum_info.StabilizerState.from_stabilizer_list` method
+    that generates a stabilizer state from a list of stabilizers.
```

Comment:
It might be good adding an example showing how this method should be used in practice (or maybe instead the function `synth_circuit_from_stabilizers`).